### ISSUE TITLE
NoClassDefFoundError with SirBlobmanCore update

### ISSUE THREAD

**Issue by SlimeDog** [SirBlobman/CombatLogX#196]
**Describe the bug**
CombatLogX failed to load when coupled with SirBlobmanCore update

**To Reproduce**
- Update SirBlobmanCore
- Start server

**Versions**
Spigot 1.16.5-3092
SirBlobmanCore version 2.2.0-b34
CombatLogX 11.0.0.0-b460

```
[18:21:43] [Server thread/INFO]: torch 0
[18:21:44] [Server thread/ERROR]: Could not load 'plugins/CombatLogX-11.0.0.0-b460.jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: java.lang.NoClassDefFoundError: com/github/sirblobman/api/core/plugin/ConfigurablePlugin
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:149) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:394) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:301) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at org.bukkit.craftbukkit.v1_16_R3.CraftServer.loadPlugins(CraftServer.java:381) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at net.minecraft.server.v1_16_R3.DedicatedServer.init(DedicatedServer.java:224) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:928) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:273) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NoClassDefFoundError: com/github/sirblobman/api/core/plugin/ConfigurablePlugin
        at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:1016) ~[?:?]
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:184) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:104) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:99) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:398) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:67) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:145) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        ... 7 more
Caused by: java.lang.ClassNotFoundException: com.github.sirblobman.api.core.plugin.ConfigurablePlugin
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:138) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:99) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        at java.lang.ClassLoader.defineClass1(Native Method) ~[?:?]
        at java.lang.ClassLoader.defineClass(ClassLoader.java:1016) ~[?:?]
        at java.security.SecureClassLoader.defineClass(SecureClassLoader.java:174) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:184) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:104) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:99) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:398) ~[?:?]
        at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:67) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:145) ~[spigot-1.16.5-3092.jar:3092-Spigot-9fb885e-ebe18b9]
        ... 7 more
```

**Comment by SlimeDog** [comment]
Issue remains with SirBlobmanCore 2.2.0-b37.

**Comment by SirBlobman** [comment]
Thanks for the report, I'll fix this soon.

**Comment by SlimeDog** [comment]
Confirmed resolved with CLX 11.0.0.0-b461. Thanks.
